package.json

```diff
@@ -5,11 +5,13 @@
   "main": "index.js",
   "scripts": {
     "start": "nodemon server",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "webpack -d --watch --mode production",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev:react": "webpack -d --watch"
```

Comment:
do you need both "build" and "dev:react" scripts? I think I understand how they may be different where build configures your webpack, but it also seems like dev:react does the same.